sdcm/nemesis.py

```diff
@@ -4013,6 +4041,14 @@ def disrupt_grow_shrink_cluster(self):
         self._grow_cluster(rack=None)
         self._shrink_cluster(rack=None)
 
+    def disrupt_grow_shrink_cluster_parallel(self):
+        sleep_time_between_ops = self.cluster.params.get('nemesis_sequence_sleep_between_ops')
+        if not self.has_steady_run and sleep_time_between_ops:
+            self.steady_state_latency()
+            self.has_steady_run = True
+        self._grow_cluster_parallel(rack=None)
+        self._shrink_cluster(rack=None)
```

Comment:
We need here the part that wait for the balanced cluster and then double the load.
During doubling the load we should also measure latency.

I think it’s more important to deal with than the parallel decommission for now.